README.md

```diff
@@ -1,6 +1,8 @@
 # Between Meals
 
-[![Build Status](https://travis-ci.org/facebook/between-meals.svg)](http://travis-ci.org/facebook/between-meals)
+[![Build Status](https://travis-ci.org/wjimenez5271/between-meals.svg)](https://travis-ci.org/wjimenez5271/between-meals)
```

Comment:
Yea I'll remove that from the PR if you like where this is going :). My intent was to get your thoughts on the concept of integrating Berkshelf.